odoo/openupgrade/doc/source/modules110-120.rst

```diff
@@ -447,7 +447,7 @@ missing in the new release are marked with |del|.
 +--------------------------------------------+-------------------------------------------------+
 |project                                     | Done                                            |
 +--------------------------------------------+-------------------------------------------------+
-|project_timesheet_holidays                  |                                                 |
+|project_timesheet_holidays                  | Nothing to do                                   |
```

Comment:
There's an openupgrade_analysis.txt file for this one.

Reply:
Yes. And? It's a nothing to do.

Reply:
You need to create the `openupgrade_analysis_work.txt` with the proper NOTHING TO DO.

Reply:
but it's already created :S

Reply:
I didn't see it... have you pushed it now? Any way, if so, we can continue when CIs finishes

Reply:
click on the 3rd commit and you should see it :S

Reply:
OK, sorry for the noise.